app/helpers/application_helper.rb

```diff
@@ -211,13 +215,17 @@ def auto_complete_search(name, val, options = {})
   end
 
   def help_path
-    link_to _("Help"), :action => "welcome" if File.exist?("#{Rails.root}/app/views/#{controller_name}/welcome.html.erb")
+    link_to(_("Help"), { :action => "welcome" }, { :class => 'btn btn-default'}) if File.exist?("#{Rails.root}/app/views/#{controller_name}/welcome.html.erb")
   end
 
   def method_path(method)
     send("#{method}_#{controller_name}_path")
   end
 
+  def hash_method_path(method)
+    send("hash_for_#{method}_#{controller_name.singularize}_path")
```

Comment:
If you can even avoid using the `hash_for_` routing helper and simply use `{ :action => 'new', :controller => controller_name }` then perhaps this starts to move away from relying on it (http://projects.theforeman.org/issues/8726)?